content/about/members.md

```diff
@@ -25,6 +25,24 @@ Naoe Tatara
 : UiO, Library, Science Library (UB)  
   _appointed by University of Oslo library_
 
-## Instructors
-
-We have many active instructors. Would you like to become an instructor, too? Join our mailing list or meet us at a study group.
\ No newline at end of file
+## Community members
+
+We have many Carpentries certified instructors, too. 
+Would you like to become an instructor, too? Join our mailing list or meet us at a study group.
```

Comment:
maybe add the mailing list address or a link where to find it?

Reply:
... perhaps also a link to the study group page of the site ` [study group]({{< relref "/study_group" >}})`

Reply:
That's true (I thought about that on Saturday and forgot today). I will fix that part and commit soon.